org.eclipse.m2e.pde.connector/build.properties

```diff
@@ -0,0 +1,5 @@
+source.. = src/
+output.. = bin/
+bin.includes = META-INF/,\
+               .,\
+               plugin.xml
```

Comment:
The `lifecycle-mapping-metadata.xml` should be added to the `bin.includes`, shouldn't it?
Otherwise it will be missing in the build jars.

Reply:
Good catch!

Reply:
Thanks.
This is a problem I occasionally notice with my colleges (and sometimes even myself 😅). Maybe there should be some mechanism in Eclipse PDE/Equinox(probably from PDE to Equinox since build.properties are from PDE) that checks if a resource is also included in the bin.includes from the build.properties.

Reply:
The problem is that this is explicitly to choose what to include and what not, so a warning or something like that has to be provided by m2e scanning for "known" extension points then check if required resources (e.g. lifecycle-mapping-metadata.xml) are present and included. the OSGI-DS plugin does this already but its always a manual step to decide if a resource has to be included or not.